index.bs

```diff
@@ -2692,6 +2780,28 @@ partial dictionary MLOpSupportLimits {
     **Returns:** an {{MLOperand}}. The output tensor that contains the result of element-wise comparison of the two input tensors.
 </div>
 
+<table id=constraints-elementwise-logical class='data' link-for="MLGraphBuilder/equal(a, b, options), MLGraphBuilder/greater(a, b, options), MLGraphBuilder/greaterOrEqual(a, b, options), MLGraphBuilder/lesser(a, b, options), MLGraphBuilder/lesserOrEqual(a, b, options), MLGraphBuilder/logicalNot(a, options)">
```

Comment:
`logicalNot` only allows "uint8" input data type. Would you like to create a separate table or say "specified as part of operation steps"?

Reply:
Oops - I meant to make that "specified...". Will fix.

Reply:
Fixed in db8d1e21ef42e7897fd7143d41dea07194dee96c